### PR TITLE
fix(r-date-range): small changes and fixies for report integration

### PR DESCRIPTION
**Pull request by qqcky**
### What was a problem?

`RGrid` has a fullscreen loader by default - that's not the common case. Added the property to be able to define it. And fixed the story.
`RDateRange` is not supporting the period as an object.

### Check lists

- [x] Readme file updated with actual description and example
- [ ] Added all ARIA attributes required for component
- [ ] Added tests for both browser and SSR render and for all components properties
- [x] Added story with all knobs and actions

### CODE REVIEW

File: packages/recomponents/src/components/r-grid/r-grid.md [hunk 1]
```diff
@@ -5,10 +5,11 @@ The `r-grid` component is designed to power the rendering of any tabular data an
...
-|--------------|------------------|---------------|-------------------------------------------------------------------------|
-| columns*     | `Object`         |               | Enumerates columns to be rendered with type specific configuration info |
-| provider*    | `Function`       |               | Returns some data to be rendered by the default scopped slot            |
+| prop               | type               | default value   | description                                                               |
+|--------------------|--------------------|-----------------| --------------------------------------------------------------------------|
+| columns*           | `Object`           |                 | Enumerates columns to be rendered with type specific configuration info   |
+| provider*          | `Function`         |                 | Returns some data to be rendered by the default scopped slot              |
+| isLoaderFullscreen | `boolean`          | `true`          | Defines if the loading state is fullscreen                                |
```
Comment (followyouhome) [inline packages/recomponents/src/components/r-grid/r-grid.md]: Please move this to comments in code

File: packages/recomponents/src/components/r-repeater/r-repeater.md [hunk 2]
```diff
@@ -5,9 +5,10 @@ This is an abstract component to help you render tabular data. It is notably use
...
 
-| prop         | type             | default value | description                                                  |
-|--------------|------------------|---------------|--------------------------------------------------------------|
-| provider*    | function         |               | Returns some data to be rendered by the default scopped slot |
+| prop               | type               | default value   | description                                                    |
+|---------------     | ------------------ | --------------- | ---------------------------------------------------------------|
+| provider*          | function           |                 | Returns some data to be rendered by the default scopped slot   |
+| isLoaderFullscreen | boolean            | `true`          | Defines if the loading state is fullscreen                     |
 
 ### Slots
```
Comment (followyouhome) [inline packages/recomponents/src/components/r-repeater/r-repeater.md]: And this too

File: packages/recomponents/src/components/r-grid/r-grid.md [hunk 3]
```diff
@@ -5,10 +5,11 @@ The `r-grid` component is designed to power the rendering of any tabular data an
...
 
 
-| prop         | type             | default value | description                                                             |
-|--------------|------------------|---------------|-------------------------------------------------------------------------|
-| columns*     | `Object`         |               | Enumerates columns to be rendered with type specific configuration info |
-| provider*    | `Function`       |               | Returns some data to be rendered by the default scopped slot            |
+| prop               | type               | default value   | description                                                               |
+|--------------------|--------------------|-----------------| --------------------------------------------------------------------------|
+| columns*           | `Object`           |                 | Enumerates columns to be rendered with type specific configuration info   |
+| provider*          | `Function`         |                 | Returns some data to be rendered by the default scopped slot              |
```
Comment (adamaltman) [inline packages/recomponents/src/components/r-grid/r-grid.md]:
```suggestion
| provider*          | `Function`         |                 | Returns some data to be rendered by the default scoped slot              |
```

**Pull request merged**